File: SistemaEscolar/FrmLogin.vb
Imports System.Data.SqlClient

Public Class FrmLogin
    Private Sub btnSair_Click(sender As Object, e As EventArgs) Handles btnSair.Click
        Dim confirma = MsgBox("Deseja Sair?", vbYesNo + vbExclamation, "Atenção")
        If confirma = vbYes Then
            Me.Close()
        End If
    End Sub

    Private Sub btnEntrar_Click(sender As Object, e As EventArgs) Handles btnEntrar.Click
        Dim usu As String = txtUsuario.Text
        Dim sen As String = txtSenha.Text

        If usu = "" Or sen = "" Then
            MsgBox("Preencha os Campos")
        Else
            Dim cmd As New SqlCommand("iniciarSessao", conexao)
            Try
                abrir()
                cmd.CommandType = 4
                With cmd.Parameters
                    .AddWithValue("@Nome", usu)
                    .AddWithValue("@Senha", sen)
                    .Add("@Msg", SqlDbType.VarChar, 100).Direction = 2
                    cmd.ExecuteNonQuery()
                End With

                usuario = txtUsuario.Text
                Dim msg As String = cmd.Parameters("@Msg").Value
                MsgBox(msg, vbInformation)

                If (msg = "Dados incorretos") Then
                    txtSenha.Clear()
                    txtUsuario.Clear()
                    txtUsuario.Focus()
                Else
                    Dim frm As New FrmPrincipal
                    Me.Hide()
                    frm.ShowDialog()
                End If


            Catch ex As Exception
                MsgBox(ex.Message)
            End Try

            conexao.Close()
        End If
    End Sub

    Private Sub btnSenha_MouseDown(sender As Object, e As MouseEventArgs) Handles btnSenha.MouseDown
        txtSenha.UseSystemPasswordChar = False

    End Sub

    Private Sub btnSenha_MouseUp(sender As Object, e As MouseEventArgs) Handles btnSenha.MouseUp
        txtSenha.UseSystemPasswordChar = True
    End Sub
End Class

File: PetShop/PetShop/FrmLogin.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Data.SqlClient;

namespace PetShop
{
    public partial class FrmLogin : Form
    {
        public FrmLogin()
        {
            InitializeComponent();
        }
        public static string usuario;
        public static string codUsuario;

        //Estabelecendo conexão com o banco de dados sql server
        SqlConnection cn = new SqlConnection(@"Data Source=DESKTOP-OJPN3JR;Initial Catalog=PetShop;Integrated Security=True;Encrypt=False");
        SqlCommand cm = new SqlCommand();

        private void btnSenha_MouseDown(object sender, MouseEventArgs e)
        {
            txtSenha.UseSystemPasswordChar = false;
        }

        private void btnSenha_MouseUp(object sender, MouseEventArgs e)
        {
            txtSenha.UseSystemPasswordChar = true;
        }

        private void btnEntrar_Click(object sender, EventArgs e)
        {
            if (txtUsuario.Text == "" || txtSenha.Text == "")
            {
                MessageBox.Show("Preencha os campos vazios", "Atenção !!", MessageBoxButtons.OK, MessageBoxIcon.Information);

            }
            else
            {
                try
                {
                    cn.Open();
                    cm.CommandText = "select * from tbl_atendente where ds_Login = ('" + txtUsuario.Text + "') and ds_Senha = ('" + txtSenha.Text + "')";
                    cm.Connection = cn;
                    SqlDataAdapter da = new SqlDataAdapter(cm);
                    DataTable dt = new DataTable();
                    da.Fill(dt);

                    if (dt.Rows.Count > 0)
                    {
                        usuario = dt.Rows[0]["nm_atendente"].ToString();
                        codUsuario = dt.Rows[0]["cd_atendente"].ToString();
                        FrmPrincipal menu = new FrmPrincipal();
                        menu.Show();
                        this.Hide();
                    }
                    else
                    {
                        MessageBox.Show("usuario ou senha invalido", "Erro !!", MessageBoxButtons.OK, MessageBoxIcon.Error);
                        txtUsuario.Clear();
                        txtSenha.Clear();
                        txtUsuario.Focus();
                    }
                }
                catch (Exception erro)
                {

                    MessageBox.Show(erro.Message);
                    cn.Close();
                }
                finally
                {
                    cn.Close();
                }
            }
        }

        private void FrmLogin_Load(object sender, EventArgs e)
        {
            txtUsuario.Focus();
        }

        private void FrmLogin_Shown(object sender, EventArgs e)
        {
            txtUsuario.Focus();
        }

        private void FrmLogin_KeyPress(object sender, KeyPressEventArgs e)
        {
            if (e.KeyChar == (char)13)
            {
                e.Handled = true;
            }
        }

        private void FrmLogin_KeyDown(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Return)
            {
                SendKeys.Send("{TAB}");
            }
        }

        private void btnSair_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }
        // SqlDataReader dt;
    }
}
